src/test/scala/carldata/series/AnalysisTest.scala

```diff
@@ -0,0 +1,43 @@
+package carldata.series
+
+import org.scalatest._
+import carldata.series.Analysis._
+
+class AnalysisTest extends FlatSpec with Matchers {
+
+  "findSessions" should "detect sessions" in {
```

Comment:
It is better to create expected Seq and just compare return value to this expected Seq.
Check how it is implemented in TimeSeriesTest